docs/vocabulary.md

```diff
@@ -388,6 +388,18 @@ pipeline in corresponding documentation.
 | **Tekton**         | N/A       | Step       | Task     | Pipeline | Trigger | Resource (?)           |
 | **Zuul**           | N/A       | N/A        | Job      | Pipeline | Trigger | Node (?)               |
 
+### Build Stages
+
+The table below is an attempt to create a mapping of common names for build stages (the unit of work one degree smaller than a pipeline). While CI/CD Tools and Technologies generally give developers broad leeway in naming and implementing their build stages, agreeing upon some common terms will help develop more pluggable build pipelines.
```

Comment:
Maybe add something like this here as well?
Some of the below steps can be found in multiple parts of the pipeline. For example it is often the case that sw linting (test step) is done prior to the build step, and deployment to a staging environment is often done prior to the release while the deployment for production often happens after the release.

Reply:
OK.